readme.md

```diff
@@ -39,6 +39,8 @@ CC0
 * [Folium](https://github.com/python-visualization/folium) - Package from creating Leaflet maps from python code.
 * [OsmApi](https://github.com/metaodi/osmapi) - Python OSM Api wrapper package.
 * [Osmcha](https://github.com/willemarcel/osmcha) - Python package to detect suspicious OSM changesets.
+* [OpenMapTiles](https://github.com/openmaptiles) - Set of tools for self-hosted vector maps and map services.
```

Comment:
Maybe change here to [Tools](https://github.com/openmaptiles/openmaptiles-tools)? And add [full site](https://openmaptiles.org/) to Full-site projects? What do you think?